src/ImageUtils.js

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2015, 2016 OpenMarket Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+'use strict';
+
+module.exports = {
+    thumbHeight: function(fullWidth, fullHeight, thumbWidth, thumbHeight) {
```

Comment:
Now that this is a public function, it would be nice to give this a comment saying what it does and what the parameters mean. How does its result relate to the `thumbHeight` param? Which of the params are allowed to be undefined? I could figure it out, but I shouldn't have to.